.azure-pipelines/gpu-tests.yml

```diff
@@ -43,6 +43,8 @@ jobs:
         lspci | egrep 'VGA|3D'
         whereis nvidia
         nvidia-smi
+        id
```

Comment:
what's one case where printing `id` would help?

Reply:
debugging the user issue
```suggestion
```